Header header
int32 ac_id
float64 voltage # in V
float64 current # in A
float64 power   # in W
float64 energy  # energy consumed in mAh
